Demo/Navigation/TurboNavigationController.swift

```diff
@@ -90,7 +90,7 @@ extension TurboNavigationController {
                 present(modalNavController, animated: animated)
             }
         } else if action == .replace {
-            let viewControllers = Array(viewControllers.dropLast()) + [viewController]
+            viewControllers = Array(viewControllers.dropLast()) + [viewController]
```

Comment:
`viewControllers` is an instance variable because we are a subclass of `UINavigationController`. We can remove line 94 if we set the property directly like you're doing.

According to [the docs](https://developer.apple.com/documentation/uikit/uinavigationcontroller/1621873-viewcontrollers):

> Assigning a new array of view controllers to this property is equivalent to calling the `setViewControllers(_:animated:)` method with the animated parameter set to `false`.

Which is exactly what we want!

Reply:
Funnily I opened my Xcode again and the original code compiles now. Anyway I opted to just rename the instance variable to `controllers` and retain the use of `setViewControllers(_:animated:)` for clarity.